### ISSUE TITLE
Unable to reproduce the results of DINO-R50-4scale-12ep (48.9 v.s. 49.2)

### ISSUE THREAD

**Issue by backseason** [IDEA-Research/detrex#137]
> > Thanks, Maybe I dont convert torchvision model to d2 model. thanks again. I will close this issue.
> 
> You're welcome, feel free to reopen it if you meet some other problems, You can also try to set the training seed to 42
> 
> ```python
> # your config
> train.seed = 42
> ```
> 
> which aligns the official training settings, this may help you to reproduce the 49.95 AP results @adamqian111

Hello, after fixing the random seed to 42, I still can not reproduce the results of dino_r50_4scale_12ep which is 49.2 in your repo. Do you have any ideas where the problem might bt?
<img width="813" alt="image" src="https://user-images.githubusercontent.com/7400760/202632285-9ff718da-f69b-46b0-bb13-cdebd585b4b8.png">

Here is the full log:
[log_detrex_dino_r50_4scale_12ep_48.87.txt](https://github.com/IDEA-Research/detrex/files/10038625/log_detrex_dino_r50_4scale_12ep_48.87.txt)

_Originally posted by @backseason in https://github.com/IDEA-Research/detrex/issues/118#issuecomment-1319588580_

**Comment by rentainhe** [comment]
> ```
>     > > Thanks, Maybe I dont convert torchvision model to d2 model. thanks again. I will close this issue.
> ```
> 
> > You're welcome, feel free to reopen it if you meet some other problems, You can also try to set the training seed to 42
> > ```python
> > # your config
> > train.seed = 42
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > which aligns the official training settings, this may help you to reproduce the 49.95 AP results @adamqian111
> 
> Hello, after fixing the random seed to 42, I still can not reproduce the results of dino_r50_4scale_12ep which is 49.2 in your repo. Do you have any ideas where the problem might bt? <img alt="image" width="813" src="https://user-images.githubusercontent.com/7400760/202632285-9ff718da-f69b-46b0-bb13-cdebd585b4b8.png">
> 
> Here is the full log: [log_detrex_dino_r50_4scale_12ep_48.87.txt](https://github.com/IDEA-Research/detrex/files/10038625/log_detrex_dino_r50_4scale_12ep_48.87.txt)
> 
> _Originally posted by @backseason in [#118 (comment)](https://github.com/IDEA-Research/detrex/issues/118#issuecomment-1319588580)_

I think 48.9AP is in the normal range, we just release our best pretrained weight using random seed. It's normal to get `48.9~49.1 AP`

**Comment by backseason** [comment]
> 

Thanks for your quick reply. Btw, how many experiments do you usually run to determine the best performing one and could you please share the full training log of the 49.2 one with me?

**Comment by rentainhe** [comment]
> > 
> 
> Thanks for your quick reply. Btw, how many experiments do you usually run to determine the best performing one and could you please share the full training log of the 49.2 one with me?

We usually run a few experiments and pick the best one

here's our dino 49.2AP training log:  [dino_r50_4scale_12ep](https://github.com/IDEA-Research/detrex/files/10053678/log.txt)

**Comment by backseason** [comment]
> > > 
> > 
> > 
> > Thanks for your quick reply. Btw, how many experiments do you usually run to determine the best performing one and could you please share the full training log of the 49.2 one with me?
> 
> We usually run a few experiments and pick the best one
> 
> here's our dino 49.2AP training log: [dino_r50_4scale_12ep](https://github.com/IDEA-Research/detrex/files/10053678/log.txt)
> 
> I think `49.0AP` is a normal result, but as an open-source toolbox, we will try our best to release better weights for the users

Many thanks! I have no more question now.

**Comment by rentainhe** [comment]
> > > > 
> > > 
> > > 
> > > Thanks for your quick reply. Btw, how many experiments do you usually run to determine the best performing one and could you please share the full training log of the 49.2 one with me?
> > 
> > 
> > We usually run a few experiments and pick the best one
> > here's our dino 49.2AP training log: [dino_r50_4scale_12ep](https://github.com/IDEA-Research/detrex/files/10053678/log.txt)
> > I think `49.0AP` is a normal result, but as an open-source toolbox, we will try our best to release better weights for the users
> 
> Many thanks! I have no more question now.

Feel free to open new issues if you meet any other questions~
